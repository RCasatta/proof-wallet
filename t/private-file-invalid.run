#!/bin/bash

$PROOFWALLET sign-psbt -m 2 -n 3 --private-file ../../t/private-invalid.txt --regtest << INPUT 2>&1
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected proofwallet.py to fail"
    exit 1
fi