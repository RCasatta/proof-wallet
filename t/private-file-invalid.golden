Error: the private-file must contain 1 line with a BIP39 mnemonic.
